lib/vtls/openssl.c

```diff
@@ -3317,7 +3317,7 @@ static CURLcode ossl_connect_step2(struct Curl_easy *data,
              error_buffer */
           strcpy(error_buffer, "SSL certificate verification failed");
       }
-#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Why is this excluding libressl? I think a comment here explaining this could be a good idea.